.github/workflows/build-install-packages.yml

```diff
@@ -0,0 +1,29 @@
+name: Build Install Packages
+
+on:
+  release:
+    types: [released]
+
+env:
+  VERSION: ${{ github.event.release.tag_name }}
+  PHP_VERSIONS: '7.3 7.4 8.0 8.1'
+  INSTALL_PACKAGES_INPUTS: '{ "flarum_version": "{0}", "php_versions": "{1}" }'
```

Comment:
Where are these magic variables coming from? Is there no named equivalent?

Reply:
it's the exact values of the env variables above it.